packages/config/package.json

```diff
@@ -49,10 +49,10 @@
     "@expo/plist": "0.0.4",
     "find-yarn-workspace-root": "^1.2.1",
     "fs-extra": "^7.0.1",
-    "globby": "^11.0.0",
+    "lodash": "4.17.15",
```

Comment:
Removed lodash from `@expo/config` here: https://github.com/expo/expo-cli/commit/fd6223ef0aad7e14c2720c82528bd77a2d9919bf (it was only used in one function)